### ISSUE TITLE
Cryfs: Volume Creation Options

### ISSUE THREAD

**Issue by pacstrap** [mhogomchungu/sirikali#31]
When creating a cryfs volume in Arch Linux, there is no option to select aes-256-gcm or aes-256-cfb from the drop-down list in Sirikali. These options can be selected if I use cryfs via the command line.

**Comment by mhogomchungu** [comment]
What is the output of the following command:

```
cryfs --show-ciphers

```

**Comment by pacstrap** [comment]
The output of the above command is...

CryFS Version 0.9.7

aes-256-gcm
aes-256-cfb
aes-128-gcm
aes-128-cfb
twofish-256-gcm
twofish-256-cfb
twofish-128-gcm
twofish-128-cfb
serpent-256-gcm
serpent-256-cfb
serpent-128-gcm
serpent-128-cfb
cast-256-gcm
cast-256-cfb
mars-448-gcm
mars-448-cfb
mars-256-gcm
mars-256-cfb
mars-128-gcm
mars-128-cfb

**Comment by mhogomchungu** [comment]
The bug is fixed in this[1] commit.

The output i am getting here is:
```
[mtz@ink ~]$ cryfs --show-ciphers
CryFS Version 0.9.7
Could not check for updates.

aes-256-gcm
aes-256-cfb
aes-128-gcm
aes-128-cfb
twofish-256-gcm
twofish-256-cfb
twofish-128-gcm
twofish-128-cfb
serpent-256-gcm
serpent-256-cfb
serpent-128-gcm
serpent-128-cfb
cast-256-gcm
cast-256-cfb
mars-448-gcm
mars-448-cfb
mars-256-gcm
mars-256-cfb
mars-128-gcm
mars-128-cfb
```

I wrote the code in SiriKali using terminal output like above without realizing that the first two lines were printed in ```std error``` and the cipher option output were printed in ```std out```. The bug in SiriKali was due to it reading only ```std out``` channel and deleting the first two entries thinking the output it got is the same as above.

This cryfs separation of its output greatly eases my concern about supporting this feature and my congratulations for cryfs developer for separating the outputs this way.

[1] https://github.com/mhogomchungu/sirikali/commit/212db7926e14443d4e936dbd4b25bf15d058893f
